hr_employee_relative/models/hr_employee_relative.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (C) 2018 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from datetime import datetime
+
+from dateutil.relativedelta import relativedelta
+
+from odoo import api, fields, models
+
+
+class HrEmployeeRelative(models.Model):
+    _name = "hr.employee.relative"
+    _description = "HR Employee Relative"
+
+    employee_id = fields.Many2one(string="Employee", comodel_name="hr.employee")
```

Comment:
No need for string and I would also remove the comodel_name, just giving as first parameter the model value